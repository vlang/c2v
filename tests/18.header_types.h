#include <stdint.h>

uint16_t ts_f32tof16(float f);
