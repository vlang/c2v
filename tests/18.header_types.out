[translated]
module tests

fn C.ts_f32tof16(f f32) u16

pub fn ts_f32tof16(f f32) u16 {
	return C.ts_f32tof16(f)
}
